draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1041,20 +1080,21 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4 - W3C Verifiable Credentials Data Model (work in progress)
+## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
 
 This example illustrates how the artifacts defined in this specification can be
-represented using W3C Verifiable Credentials Data Model as defined in
-[@VC_DATA].
+represented as W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] without using JSON-LD.
+It uses a content type `credential-claims-set+json` defined in [@VC_JWT], Section 3.
 
-SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a VC.
+SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a W3C VC.
 
-A Presentation with a Holder Binding JWT is equivalent to a Holder-signed W3C Verifiable Presentation (W3C VP).
+A Holder-signed Verifiable Presentation as defined in [@VC_DATA_v2.0] would be equivalent to
```

Comment:
"would be" or "is"?

Reply:
`would be` because we are not putting the example of a VP.

Reply:
will be with an issue marker ?

Reply:
```suggestion
A Holder-signed Verifiable Presentation as defined in [@VC_DATA_v2.0] is equivalent to
```